### DISCUSSION TITLE
Brain DICOMs and contrasts

### DISCUSSION THREAD

**Discussion by nicolasblumenroehr** [facebookresearch/fastMRI#287] [General]
Dear community,

in the paper "fastMRI: An Open Dataset and Benchmarks for Accelerated MRI" is a section that describes the brain DICOM dataset.  However, in the entire data set are only the DICOM knees images included, the ones for the brains are missing, or at least they are not provided separately. Can they be acquired from somewhere else, or are they part of the other data sets, i.e. train, val and test batches?

Another question is about the contrasts. The data set description provides information about the contrasts the data was measured with. There are also some examples in the papers for different contrasts of the same image. I wonder, if there is any chance the information about the contrast for each image (label) is provided in the data set, or somewhere else? For example, as part of the DICOM header and if yes, which attribute is that?

Thanks a lot in advance!

**Comment by mmuckley** [comment]
Hello @nicolasblumenroehr, there is a brain DICOM set, but in the shuffle converting to version 2.0 of the data we forgot to include the link in the email. I've messaged the NYU team that manages the account to add it back in. I'll post here when I can get confirmation that it's complete.

Getting the contrast can be a little tricky. Technically, all MRI images are on a continuum of contrast. What we specify in practice are pulse sequence parameters like TE and TR that can weigh the images in different ways.

There are a few places in headers where you can extract intended contrast, the main ones of which will be any sequence name that the techs put into the system when they ran the scan. There can be some inconsistencies (e.g., maybe on person puts "T2" and another puts "AXT2", etc.), but you can probably filter out most of the inconsistencies with a little bit of data wrangling.

**Reply by nicolasblumenroehr** [reply]
Many thanks!

**Comment by mmuckley** [comment]
Hello @nicolasblumenroehr, the DICOMs should be available now (in legacy `tar.gz` format).

**Reply by branme96** [reply]
Hi @mmuckley, sorry if this isn't the appropriate place to ask.

I've been trying to access the DICOM dataset, but the .tar.gz file seems to be corrupted. gzip gets a CRC error and tar can only unzip about 5.6gb.

I've been using tar -zxvf brain_fastMRI_DICOM.tar.gz with no luck.

Let me know if you need anything from me.

Thanks

**Reply by mmuckley** [reply]
Hello @branme96, did you verify that the SHA hash matches the one that you can see from AWS? First we need to verify that the corruption didn't happen during download.

**Reply by branme96** [reply]
Hi @mmuckley, thanks so much for your response!

I have downloaded the dataset from my personal machine using Firefox and on another machine using the "curl" command recommended in the email. Both of these methods yielded the same SHA hash, but as you suspected they were both different to the hash on AWS.

Do you think this could still be an issue on my end?

**Reply by mmuckley** [reply]
Hello @branme96, I have messaged the NYU group that manages the AWS account that has the data. You may also try messaging them at [fastmri@med.nyu.edu](https://fastmri.med.nyu.edu/fastmri@med.nyu.edu). I do not manage the data myself.

**Reply by branme96** [reply]
Hi @mmuckley, thanks again. I really appreciate your help!
